src/traces/image/plot.js

```diff
@@ -66,11 +76,14 @@ module.exports = function plot(gd, plotinfo, cdimage, imageLayer) {
         }
 
         // Reduce image size when zoomed in to save memory
-        var extra = 0.5; // half the axis size
-        left = Math.max(-extra * xa._length, left);
-        right = Math.min((1 + extra) * xa._length, right);
-        top = Math.max(-extra * ya._length, top);
-        bottom = Math.min((1 + extra) * ya._length, bottom);
+        if(!fastImage) {
+            var extra = 0.5; // half the axis size
```

Comment:
You may drop this variable.

Reply:
> You may drop this variable.

I'm sorry but which one?

Reply:
The variable titled `extra`.

Reply:
Are you suggesting that I hardcode `0.5` in the following few lines? @archmoj

Reply:
> Are you suggesting that I hardcode `0.5` in the following few lines? @archmoj

Yes please.

Reply:
Personally, I would leave it as is in case we want to change the value in the future. @alexcjohnson what do you think?

Reply:
OK resolving for now.